src/styles/block/stopwatch.scss

```diff
@@ -0,0 +1,52 @@
+$stopwatch-size: 80vmin;
+
+.stopwatch {
+  box-sizing: border-box;
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: $stopwatch-size;
+  width: $stopwatch-size;
+  border: 0.02 * $stopwatch-size dotted darkblue;
+  border-radius: 50%;
+
+  &__center {
+    position: absolute;
+    height: 0.08 * $stopwatch-size;
+    width: 0.08 * $stopwatch-size;
+    border-radius: 50%;
+    background-color: #000;
+  }
+
+  &__hand-second {
+    height: 50%;
+    width: 2%;
+    background-color: yellow;
+    position: absolute;
+    bottom: 50%;
+    animation: rotate 5s infinite linear;
+    transform-origin: 50% 100%;
+    border-radius: 50%;
+  }
+
+  &__hand-minute {
+    height: 35%;
+    width: 1%;
+    background-color: darkblue;
+    position: absolute;
+    bottom: 50%;
+    animation: rotate 60s infinite linear;
+    transform-origin: 50% 100%;
+    border-radius: 50%;
+  }
```

Comment:
these two hands have a lot of the same css rules. You should follow DRY (Don't Repeat Yourself) principle

Either use one more common selector `.stopwatch__hand` where you move common styles with modifiers `.stopwatch__hand--minute`/`.stopwatch__hand--hour` or create placeholder/mixin.